config/templates/sprint_general_label_layout.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Oh gosh another place with Printing Label templates 😵‍💫 ! Do they also get added to the deployment project? Or were there any alternatives locations for their storage, sprint/ PMB/ lighthouse/ ss?